### PR TITLE
mpv: update to 0.26.0

### PR DESCRIPTION
**Pull request by cr6git**

### CODE REVIEW

File: srcpkgs/mpv/template [hunk 1]
```diff
@@ -18,11 +18,11 @@ makedepends="
  libXv-devel libxkbcommon-devel libva-glx-devel libvdpau-devel alsa-lib-devel
  pulseaudio-devel libbluray-devel libcdio-paranoia-devel libdvdread-devel
  MesaLib-devel harfbuzz-devel libXScrnSaver-devel jack-devel libdvdnav-devel
- wayland-devel libwayland-egl libuuid-devel libguess-devel libXrandr-devel samba-devel
```
Comment (Gottox) [inline srcpkgs/mpv/template]: Why was samba removed?

**Comment by chneukirchen** [comment]
In any case, samba makes mpv GPL-3.

Is there a single user that needs the samba dependency?

File: srcpkgs/mpv/template (hunk 1 above)
Comment (cr6git) [inline srcpkgs/mpv/template]: Samba was already disabled by default... 

`Checking for Samba support (makes mpv GPLv3)                      : disabled`

**Comment by Gottox** [comment]
Don't think so.

Our samba is still gplv2 iirc.

**Comment by cr6git** [comment]
@Gottox GPL-2 ? The [samba template](https://github.com/voidlinux/void-packages/blob/master/srcpkgs/samba/template#L18) is GPL-3.

**Comment by Gottox** [comment]
hmm... true. anyway, the change is fine.

**Pull request merged**